protos/v1alpha2/team_link.proto

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/team-link/apis/v1alpha2";
```

Comment:
Just checking that you meant to omit 'abcxyz' from this link; probably intentional.

Reply:
Actually I think it should have `abcxyz` in the path. Thanks for the catch!